src/test/java/core/basesyntax/OperationHandlerImplTest.java

```diff
@@ -0,0 +1,101 @@
+package core.basesyntax;
+
+import static core.basesyntax.model.FruitTransaction.Operation.BALANCE;
+import static core.basesyntax.model.FruitTransaction.Operation.PURCHASE;
+import static core.basesyntax.model.FruitTransaction.Operation.RETURN;
+import static core.basesyntax.model.FruitTransaction.Operation.SUPPLY;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.hadler.OperationHandler;
+import core.basesyntax.hadler.impl.BalanceOperationHandler;
+import core.basesyntax.hadler.impl.DecreaseOperationHandler;
+import core.basesyntax.hadler.impl.IncreaseOperationHandler;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class OperationHandlerImplTest {
+    private OperationHandler operationHandler;
```

Comment:
```suggestion
    private OperationHandler balanceHandler;
    private OperationHandler increaseHandler;
    private OperationHandler decreaseHandler;
```
you can move handlers here, and initialize them in @Before method, use each handler whereever you need